web/source/kmwuifloat.ts

```diff
@@ -126,7 +126,7 @@ if(!window['keyman']['ui']['name']) {
       ui.innerDiv.appendChild(ui.KeyboardSelector);  //this may need to be moved up.... 
 
       // Check required interface alignment and default keyboard
-      var opt=util['getOption']('ui'),dfltKeyboard='English';
+      var opt=util['getOption']('ui'),dfltKeyboard='(System keyboard)';
```

Comment:
Can `value` in l. 240 get removed too?
https://github.com/keymanapp/keyman/blob/0093c8b991d402bc34f2ed54da6a30b3c1d8a16c/web/source/kmwuifloat.ts#L240